js/index.js

```diff
@@ -0,0 +1,38 @@
+console.clear();
+
+const showAnswerButton1 = document.querySelector('[data-js="show-answer1"]');
+const answer1 = document.querySelector('[data-js="answer1"]');
+const showAnswerButton2 = document.querySelector('[data-js="show-answer2"]');
+const answer2 = document.querySelector('[data-js="answer2"]');
+const showAnswerButton3 = document.querySelector('[data-js="show-answer3"]');
+const answer3 = document.querySelector('[data-js="answer3"]');
+
+showAnswerButton1.addEventListener("click", () => {
+  if (showAnswerButton1.textContent == "Show Answer") {
+    showAnswerButton1.textContent = "Hide Answer";
```

Comment:
Right now I think this is saying that if your button says "Show answer", the buttons content is supposed to change to hide answer, I'm not sure if this is what you wanted